src/BizHawk.Client.EmuHawk/MainForm.Designer.cs

```diff
@@ -17,2407 +17,2407 @@ partial class MainForm
 		/// </summary>
 		private void InitializeComponent()
 		{
-			this.components = new System.ComponentModel.Container();

```

Comment:
I still want don't want this checked in. If `git diff -w filename` is blank, don't stage the file.